etcd/install.sls

```diff
@@ -9,17 +9,10 @@ include:
 
   {%- if etcd.manage_users %}
 etcd-user-group-home:
-  group.present:
-    - name: {{ etcd.group or 'etcd' }}
-    - system: True
-    - require_in:
-      - user: etcd-user-group-home
   user.present:
     - name: {{ etcd.user or 'etcd' }}
-    - gid_from_name: True
+    - gid: {{ etcd.user or 'etcd' }}
```

Comment:
Based on @noelmcloughlin's [comment](https://github.com/saltstack-formulas/etcd-formula/pull/40/files#r456573876), it looks like this should be:

```suggestion
    - gid: {{ etcd.group or 'etcd' }}
```